src/features/DisplayControls/NursingTasks/components/UpdateNursingTasks.jsx

```diff
@@ -405,12 +450,11 @@ const UpdateNursingTasks = (props) => {
                           onChange={(time) => {
                             handleTimeChange(time, medicationTask.uuid);
                           }}
-                          labelText={
-                            enable24HourTime
-                              ? `Task Time (${timeText24})`
-                              : `Task Time (${timeText12})`
-                          }
-                          invalidText={invalidTimeText}
+                          labelText={`Task Time (${timeText24})`}
```

Comment:
Don't we need 24 hr checks here?

Reply:
Not required as we have a check before for 24hours time and based on that we render two different component